### PR TITLE
added display, change operations

### PR DESCRIPTION
**Pull request by mehak2323**
Added the following stack operations:
-display() it prints all the elements available in the stack
-change(index,data) it changes the element at the given position
-getmin() returns the minimum element from Stack
and updated the test case. #89

### CODE REVIEW

File: Code/C++/stack_class.cpp [hunk 1]
```diff
@@ -80,39 +126,37 @@ int main()
...
-    }
-  return 0;
+    return 0;
 }
 
-
-/* 
-
+/*
 	Test Case :
```
Comment (rudrakshi99) [inline Code/C++/stack_class.cpp:139]: add at least 2 test cases.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by mehak2323** [comment]
I've added a test case, please tell me any further changes. And when to update readme.

**Comment by tarun26091999** [comment]
@rudrakshi99 getmin() was not specified in the issue.

**Comment by rudrakshi99** [comment]
@mehak2323 Please remove getmin() function from your code as you have not mentioned it in your issue.

**Comment by mehak2323** [comment]
I have removed the getmin() function code now. Please let me know what I have to do next.

**Comment by rudrakshi99** [comment]
I think you have not removed it. Where is the commit?

**Comment by mehak2323** [comment]
Please check now.

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**